pkg/config/services.go

```diff
@@ -17,9 +17,10 @@ import (
 type ServicesConfig struct {
 	ServicePrincipalMapping map[string]string `mapstructure:"principal_mapping" flag:"service-principal-mapping" toml:"principal_mapping,omitempty"`
 
-	Indexer   IndexingServiceConfig  `mapstructure:"indexer" validate:"required" toml:"indexer,omitempty"`
-	Upload    UploadServiceConfig    `mapstructure:"upload" validate:"required" toml:"upload,omitempty"`
-	Publisher PublisherServiceConfig `mapstructure:"publisher" validate:"required" toml:"publisher,omitempty"`
+	Indexer       IndexingServiceConfig       `mapstructure:"indexer" validate:"required" toml:"indexer,omitempty"`
+	EgressTracker EgressTrackingServiceConfig `mapstructure:"etracker" validate:"required" toml:"etracker,omitempty"`
```

Comment:
See comment here regarding making this a required config: https://github.com/storacha/piri/pull/243/files#r2353386178 - I think it should be optional.